.github/workflows/ci.yml

```
@@ -48,22 +48,15 @@ jobs:
         - use-conda: 'Yes'  # No conda packages yet for Qt6
           skip-pyqt6: true
           skip-pyside6: true
-        - python-version: '3.6'
-          use-conda: 'No'
-          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.6 defaults channel
-          pyside2-version: '5.12.0'  # 5.12.1-5.12.6 fails on collection/segfaults on patch test
         - os: windows-latest
           python-version: '3.10'
           use-conda: 'No'
           pyside2-version: '5.15'  # No 5.12 wheel on Windows and Python 3.10
         - os: windows-latest
-          python-version: '3.6'
+          python-version: '3.7'
           use-conda: 'Yes'
-          pyqt5-qt-version: '5.9'  # 5.12 is apparently unreliable here
+          pyqt5-qt-version: '5.9'
         - os: macos-latest
```

This was still left in here somehow. However, seems like the test runner [crashes early in startup](https://github.com/spyder-ide/qtpy/runs/5726345093?check_suite_focus=true) on macOS + Py3.7 + PySide6 (+ conda=no), apparently due to an issue with `pytest-qt` (possibly some interaction with macOS 11+?). I can't remember if this was the same issue as before, but unless you know of a better solution, we should add back

```suggestion
        - os: macos-latest
          python-version: '3.7'
          use-conda: 'No'
          skip-pyside6: true  # pytest-qt crashes on pytest startup under this environment
```